File: jeonghyeon/3week/아이스 아메리카노.js
function solution(money) {
    var answer = [];
    answer[0] = parseInt(money / 5500); // 나눈 값의 정수 부분만 추출해 배열에 넣음
    answer[1] = money % 5500; // 남는 돈은 나머지로 구해서 배열에 넣음
    return answer;
}

File: gang/week3/아이스 아메리카노.java
class Solution {
    public int[] solution(int money) {
        int[] answer = new int[2];
        answer[0] = (money / 5500); // 금액 / 아메리카노 가격해서 잔수 구함
        answer[1] = (money % 5500); // 금액 % 아메리카노 가격 해서 잔돈 구함
        return answer;
    }
}